ts/src/decoding/fastPforSpec.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Shared constants and helpers for the FastPFOR codec.
+ */
+
+export type Int32Buf = Int32Array<ArrayBufferLike>;
+export type Uint8Buf = Uint8Array<ArrayBufferLike>;
+
+/**
+ * Bit masks for each bitwidth 0-32.
+ * DO NOT MUTATE - this is a shared constant.
+ */
+export const MASKS: Readonly<Uint32Array> = (() => {
```

Comment:
I don't understand why this is defined and then executed as a method instead of saving the relevant size/array/constant... Can you elaborate why you chose to use this technic?